src/libdrakvuf/libdrakvuf.h

```diff
@@ -223,6 +223,34 @@ struct drakvuf_trap {
     void *data;
 };
 
+
+////////////////////////////////////////////////////////////////////////////
+
+// IMHO these definitions must be placed within another file, named
+// libdrakvuf-windows.h or something similar
+
+// For get_previous_mode...
+typedef enum _MODE {
+    KernelMode,
+    UserMode,
+    MaximumMode
+} MODE ;
+
+// Confirmed only on Win7 SP1...
+typedef enum _OBJECT_MANAGER_OBJECT_TYPE {
+    ObjManagerProcessObject = 7,
+    ObjManagerThreadObject  = 8
+} OBJECT_MANAGER_OBJECT_TYPE ;
```

Comment:
And the typedef enums are lower-case, with the type at the end being signified by _t, so this would be object_manager_object_t